src/Microsoft.AspNetCore.Mvc.TagHelpers/FormActionTagHelper.cs

```diff
@@ -101,6 +107,12 @@ public FormActionTagHelper(IUrlHelperFactory urlHelperFactory)
         public string Area { get; set; }
 
         /// <summary>
+        /// The URL fragment name.
```

Comment:
"Gets or sets the URL fragment."

Seems this class is missing "Gets or sets" in general. @Eilon should @jbagga change things to use that approach consistently or leave it out here for consistency w/ other comments. In any case, this property is definitely not a fragment _name_.